packages/components/src/popover/style.scss

```diff
@@ -171,7 +171,12 @@ $arrow-size: 8px;
 
 	.components-popover:not(.is-mobile):not(.is-middle).is-right & {
 		/*!rtl:ignore*/
-		margin-left: -24px;
+		margin-left: 0;
+	}
+
+	.components-popover.edit-post-more-menu__content:not(.is-mobile):not(.is-middle).is-right & {
+		/*!rtl:ignore*/
```

Comment:
Could you add some comments explaining why these should be ignored? Maybe even just a link to this PR, just so future developers will understand.

Reply:
`*!rtl:ignore*/` came from the CSS statement before, which I copied and adjusted.